parsers/nmparser/parse_run_details.go

```diff
@@ -101,27 +87,11 @@ func ParseRunDetails(lines []string) RunDetails {
 		}
 	}
 
-	if version == "7.4.3" {
-		if runStart == "" {
-			runStart = lines[0]
+	if runDetails.Version == "7.4.3" {
```

Comment:
too specific, should just be if nonmem 7.4 vs 7.3. What will happen when nonmem 7.4.4 comes out

Reply:
ok (that LOC is pretty old)